ted/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -664,20 +668,17 @@ public void zeroRobotPosition()
    * @param visionMeasurement the most recent vision measurement provided by vision subsystem
    */
   private void addVisionMeasurement(VisionMeasurement visionMeasurement){
-    // The wpilib matrix constructor requires sizes specified as Nat types. 
-    // trying to update the matrix dynamicaly leads to issues
-    Matrix<N3,N1> visionStdDev = MatBuilder.fill(Nat.N3(), Nat.N1(), new double[]{0.9, 0.9, 0.9});
-    // for now ignore all vision measurements that are null or contained robot position is null
     // for now ignore all vision measurements that are null or contained robot position is null
     if (visionMeasurement != null && useVision){
       Pose2d visionComputedMeasurement = visionMeasurement.getRobotPosition();
       if(visionComputedMeasurement != null) {
-          swervePoseEstimator.addVisionMeasurement(visionComputedMeasurement, visionMeasurement.getTimestamp(), visionStdDev);
+        if(visionComputedMeasurement.getTranslation().getDistance(getRobotPosition().getTranslation()) <= 1){
```

Comment:
Add a comment that we want to reject vision estimates that are more than 1 meter from where the odometry thinks it is. Based on advice here: 
https://www.chiefdelphi.com/t/swerve-drive-pose-estimator-and-add-vision-measurement-using-limelight-is-very-jittery/453306

Reply:
added